src/main/kotlin/no/nav/familie/klage/integrasjoner/FagsystemVedtakService.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.klage.integrasjoner
```

Comment:
Usikker på om dette blir riktig pakke for denne? Noen forslag?

Reply:
`no.nav.familie.klage.fagsystem` ?